baseplate/integration/pyramid.py

```diff
@@ -17,6 +17,11 @@ def make_app(app_config):
 
         return configurator.make_wsgi_app()
 
+.. warning::
+
+    You should never set an exception handler for the base ``Exception``. This
```

Comment:
I think this needs something a bit more concrete. Questions someone might think while reading this for the first time: What's an exception handler? (they appear to be called "exception views" in pyramid docs) what's a baseplate observer? (i.e. what does that actually mean to me as an application dev)

Maybe:

```
Because of how Baseplate instruments Pyramid, you should not 
make an `exception view`_ that handles the base :py:exc:`Exception`
as this will prevent Baseplate from seeing the unhandled error
and reporting it appropriately.

.. exception view: https://docs.pylonsproject.org/projects/pyramid_cookbook/en/latest/pylons/exceptions.html
```

Reply:
Oh yea, I like that a lot more. Thanks for the catch.

Reply:
take my rST syntax with a grain of salt though :)

Reply:
Ha yea I did :P